util/pom.xml

```diff
@@ -24,6 +24,19 @@
 			<artifactId>commons-codec</artifactId>
 			<version>1.10</version>
 		</dependency>
+		<!-- test dependencies -->
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/com.github.stefanbirkner/system-rules -->
+		<dependency>
```

Comment:
The dependency provides the `EnvironmentVariables` class used here:

https://github.com/kubernetes-client/java/pull/17/files#diff-f4d44c285e36e2ee7e3edfe79246b2f7R22